Sources/docc/DocCDocumentation.docc/linking-to-symbols-and-other-content.md

```diff
@@ -0,0 +1,198 @@
+In this scenario, and to ensure uniqueness, DocC uses the symbol's unique 
+identifier instead of its name to disambiguate. DocC's warnings about ambiguous
+symbol links suggests one disambiguation for each of the symbols that match the
+ambiguous symbol path.
+
+```markdown
+### Updating Sloths
+- ``Sloth/update(_:)-4ko57``
+- ``Sloth/update(_:)-jixx``
+```
+
+In the example above, both symbols are functions so you need the unique 
+identifiers to disambiguate the `Sloth/update(_:)` link. 
+
+Unique identifiers aren't the only way to disambiguate symbol links. If a symbol
+has a different type from the other symbols with the same symbol path, you can 
+use that symbol's type suffix to disambiguate the link and make the link refer 
+to that symbol. For example, consider a `Color` structure with `red`, `green`, 
+and `blue` properties for color components and static properties for a handful 
+of predefined color values.
+
+```swift
+public struct Color {
+    public var red, green, blue: Double
+}
+
+extension Color {
+    public static let red    = Color(red: 1.0, green: 0.0, blue: 0.0)
+    public static let purple = Color(red: 0.5, green: 0.0, blue: 0.5)
+    public static let blue   = Color(red: 0.0, green: 0.0, blue: 1.0)
+}
+```
+
+Both the `red` property and the `red` static property have a symbol path of 
+`Color/red`. Because these are different types of symbols you can disambiguate 
+`Color/red` with symbol type suffixes instead of the symbols' unique identifiers.
+
+The following example shows a symbol link to the `red` property:
+
+```markdown
+``Color/red-property``
+```
+
+The following example shows a symbol link to the `red` static property:
+
+```markdown
+``Color/red-type.property``
+```
+
+DocC supports the following symbol types for use in symbol links:
+
+| Symbol type       | Suffix            |
+|-------------------|-------------------|
+| Enumeration       | `-enum`           |
+| Enumeration case  | `-enum.case`      |
+| Protocol          | `-protocol`       |
+| Operator          | `-func.op`        |
+| Typealias         | `-typealias`      |
+| Function          | `-func`           |
+| Structure         | `-struct`         |
+| Class             | `-class`          |
+| Type property     | `-type.property`  |
+| Type method       | `-type.method`    |
+| Type subscript    | `-type.subscript` |
+| Property          | `-property`       |
+| Initializer       | `-init`           |
+| Deinitializer     | `-deinit`         |
+| Method            | `-method`         |
+| Subscript         | `-subscript`      |
+| Instance variable | `-ivar`           |
+| Macro             | `-macro`          |
+| Module            | `-module`         |
+
+Symbol type suffixes can include a source language identifier prefix — for 
+example,  `-swift.enum` instead of `-enum`. However, the language 
+identifier doesn't disambiguate the link.
+
+Symbol paths are case-sensitive meaning that symbols with the same name in
```

Comment:
```suggestion
Symbol paths are case-sensitive, meaning that symbols with the same name in
```

Would be good to provide an example of this here.

Reply:
Resolved https://github.com/apple/swift-docc/pull/742/commits/2e64258bcfa7eff3e8272bf1c2075245b5f4d9c5